lib/model.js

```diff
@@ -292,6 +292,11 @@ Model.prototype.$__handleSave = function(options, callback) {
 
     const delta = this.$__delta();
 
+    const useMongoTimestamp = this.schema.options.useMongoTimestamp;
+    if (useMongoTimestamp) {
+      delta[1]['$currentDate'] = { updatedAt: true };
```

Comment:
This is probably not correct since the user defined key may be different than `updatedAt` I suppose.